### PR TITLE
Strings reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: Monal/Base.lproj/Localizable.strings [hunk 1]
```diff
@@ -17,7 +17,7 @@
 "Accept" = "Accept";
 
 /* No comment provided by engineer. */
-"Access message archives on the server." = "Access message archives on the server.";
+"Access message archives on the server." = "Access messages are archived on the server.";
```
Comment (licaon-kter) [inline Monal/Base.lproj/Localizable.strings:20]: we "access" the "message archive"... it's a XMPP thing, the original was correct.

Comment (comradekingu) [reply Monal/Base.lproj/Localizable.strings:20]: Who is the "we" there. I am thinking this could be a whole lot easier to translate if that was clear.

Comment (Echolon) [reply Monal/Base.lproj/Localizable.strings:20]: Eventually turn the structure to: Archive for access messages is on the server.

?

Comment (licaon-kter) [reply Monal/Base.lproj/Localizable.strings:20]: We the user of the app...

**Comment by licaon-kter** [comment]
@anurodhp @tmolitor-stud-tu What is this about?

`"string here" = "string here";`
...this PR should modify both, right? And the right side would be translated, I guess

**Comment by tmolitor-stud-tu** [comment]
@licaon-kter the left side is the string as used in code and **must not be changed**.

**Comment by tmolitor-stud-tu** [comment]
"`Access message archive on the server`" would be correct

**Comment by Echolon** [comment]
Shall I close?

File: Monal/Base.lproj/Localizable.strings [hunk 2]
```diff
@@ -17,7 +17,7 @@
 "Accept" = "Accept";
 
 /* No comment provided by engineer. */
-"Access message archives on the server." = "Access message archives on the server.";
+"Access message archives on the server." = "Access your message archives on the server.";
```
Comment (comradekingu) [inline Monal/Base.lproj/Localizable.strings:20]:
```suggestion
"Access message archives on the server." = "Access message archive on the server.";
```

**Comment by comradekingu** [comment]
@Echolon Why not merge it? Looks good to me. Went with @tmolitor-stud-tu's suggestion.

**Comment by tmolitor-stud-tu** [comment]
This changes should be done in the code, not in the translation.
Doing them in the translation will make it very complicated to change the code later on.

**Comment by tmolitor-stud-tu** [comment]
--> check the code for each of these strings and change them there. then make a new pull request with these changes.

**Pull request closed without merging**